priv/riak_kv.schema

```diff
@@ -967,3 +967,45 @@
   fun(Value) when is_integer(Value) andalso Value > 0 -> true;
      (_) -> false
   end}.
+
+%% @doc Register allowable origins (for CORS), specified as a list of ip
```

Comment:
Talked with @paegun about the schema today.  I'm cool with `cors_allowable_origin` since there aren't any other CORS settings (and don't appear to be the possibility of more)